.github/pull_request_template.md

```diff
@@ -30,5 +30,4 @@ that your change is to big and it is suggested to split it into several smaller
 - [ ] The documentation has been adjusted accordingly
 - [ ] The changes generate no new warnings
```

Comment:
I always find this line awkward. If there has to be something to warn about, we should probably do it.
Same for the "self review" line. If one has not looked at the published code, this line will not change much about that

Reply:
removed it